### ISSUE TITLE
Static Help Names

### ISSUE THREAD

**Issue by jeremyredhead** [jedevc/botrulez#5]
Some bots have a changing display name, in order to provide some sort of information. For example, Garmy's infobot, POMBot, embers/firebot, that one that shows the time (Xyzzy's?). However, not all of these respond to commands using their "proper names" (eg `!help @POMBot`), which may can make usage annoying or difficult, especially on mobile or if the display is changing frequently. 

Thus I propose adding some sort of rule or guideline about "static names" to the "Non-standard commands" section, eg "bots whose display name regularly changes should have a "static name" which they also respond to as well as their current one, and which is somehow mentioned in their !help"

Thoughts, feedback?

**Comment by CylonicRaider** [comment]
Sounds good to me!

**Comment by iykdw** [comment]
Agreed; it’s probably especially useful for bots whose names change frequently - for instance, POMbot is relatively unlikely to change name between its nick being autocompleted and ‘enter’ being pressed, but the time bot and firebot especially are important. I think it would be worth specifying all nicks a bot responds to in the response to !help.

**Comment by CylonicRaider** [comment]
If the help does not list nicknames, one should assume the current name of the bot.

**Comment by Garmelon** [comment]
If the static nick was shown in the general `!help`, it would be reachable by mobile.
How about the format:
`[(10P15B1N)] @PBL | I count the types of clients in my nick`

**Comment by CylonicRaider** [comment]
I'd perhaps append a `(see !help @PBL)` instead.
